### DISCUSSION TITLE
SMTP SSL

### DISCUSSION THREAD

**Discussion by gajosew** [F43nd1r/Acrarium#329] [Ideas]
Are there any plans for smtp ssl support? 
Not only with port 25 open, but also with 587

**Comment by F43nd1r** [comment]
Should already be possible. Check the spring boot documentation to find out which properties need to be set.

**Reply by gajosew** [reply]
Ok
________________________________
spring.mail.host=smtp.gmail.com
spring.mail.port=587
spring.mail.username=username
spring.mail.password=password

# Other properties
spring.mail.properties.mail.smtp.auth=true
spring.mail.properties.mail.smtp.connectiontimeout=5000
spring.mail.properties.mail.smtp.timeout=5000
spring.mail.properties.mail.smtp.writetimeout=5000

# TLS , port 587
spring.mail.properties.mail.smtp.starttls.enable=true
_______________________________________________

but 
acrarium send email as root@id_container

 Recipient address rejected: 2.server.smtp is not allowed to send from root@82ea6032536f or from root@82ea6032536f
